docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -267,87 +310,84 @@ Congratulate and counsel the mother on appropriate IYCF practices.
 
-  - Effective coverage ('met-need') = :math:`\text{treatment coverage}\times\text{treatment efficacy}`
+  - Effective coverage (EC) = :math:`\text{treatment coverage}\times\text{treatment efficacy}`
 
   - Not effectively covered = 1 - effective coverage
 
-For example, a service with treatment coverage of 50%, and a cure-rate of 90% is meeting 50% x 90% = 45% of need (effective coverage of 45%). 
-
 .. image:: effective_coverage_figure.svg
 
-
-| SAM programme treatment coverage: 48.8% (37.4 - 60.4) (this is a point coverage; assumes programmes are not good at case finding) [Isanaka_2021]_
+| SAM programme treatment coverage: 48.8% (37.4-60.4) (this is a point coverage; assumes programmes are not good at case finding) [Isanaka_2021]_
 | MAM programme treatment coverage: same as SAM for now until this website is updated https://acutemalnutrition.org/en/countries 
 
 
-.. note::
-
-  1) Add treatment efficacy for SAM and MAM programmes 
-  2) Discuss how coverage is estimated (SQUEAC surveys). Discuss difference in point-coverage and period-coverage. 
-  3) Update coverage values to 'effective coverage' values
-
+.. todo::
 
+  Discuss how coverage is estimated (SQUEAC surveys). Discuss difference in point-coverage and period-coverage. 
+ 
 
 .. list-table:: Baseline coverage data
-  :widths: 10 10 5 15 20
+  :widths: 10 10 5 15 20 20
   :header-rows: 1
 
   * - Location
     - Subpopulation
     - Coverage
-    - Value
+    - Value (CxE)
+    - Reference
     - Note
-  * - Ethiopia
-    - Tx for infants 0-6 months with MAM 
-    - :math:`C_{MAM_{0-6}}`
-    - 48.8% (37.4 - 60.4) for now  
-    - Update this value to post-natal care coverage
   * - Ethiopia
     - Tx for infants 0-6 months with any SAM 
-    - :math:`C_{SAM_{0-6}}`
-    - 48.8% (37.4 - 60.4) for now  
-    - Update this value to post-natal care coverage
+    - :math:`EC_{SAM_{0-6}}`
+    - 48.8%(95%CI:37.4-60.4) x 70%(95%CI:64-76)
+    - [Isanaka_2021]_ [Zw_2020tx]_
+    - Update C, E values for infants?
   * - Ethiopia
-    - Tx for kids 6-59 months with MAM 
-    - :math:`C_{MAM_{6-59}}`
-    - 48.8% (37.4 - 60.4) ~ stand-in until we find updated data
-    - Coverage of TSFP for kids 6-59 months with MAM
+    - Tx for children 6-59 months with SAM
+    - :math:`EC_{SAM_{6-59}}` 
+    - 48.8%(95%CI:37.4-60.4) x 70%(95%CI:64-76)
+    - [Isanaka_2021]_ [Zw_2020tx]_
+    - 
+  * - Ethiopia
+    - Tx for infants 0-6 months with MAM 
+    - :math:`EC_{MAM_{0-6}}`
+    - 48.8%(95%CI:37.4-60.4) x 73.1% (RUSF)
+    - [Isanaka_2021]_ [Ackatia_Armah_2015tx]_
+    - Update C and E value for MAM
   * - Ethiopia
-    - Tx for kids 6-59 months with SAM
-    - :math:`C_{SAM_{6-59}}` 
-    - 48.8% (37.4 - 60.4)
-    - Coverage of SC or OTP for SAM kids
+    - Tx for kids 6-59 months with MAM 
+    - :math:`EC_{MAM_{6-59}}`
+    - 48.8%(95%CI:37.4-60.4) x 73.1% (RUSF)
```

Comment:
oh, and Rajan also asked for the distribution from which to draw values, which could be as simple as saying `normal(mean, (ub-lb)/(2*1.96) `

Reply:
done!